demo/README.md

```diff
@@ -0,0 +1,23 @@
+# VM Definitions for vagrant 
+
+## adding new box 
+
+to add a new demo add a new folder with following structure
+
+```
+vmdefs/enabled/centos6-drools-guest/
+├── ansible.cfg
+├── playbook.yml
+└── Vagrantfile
+```
+
+## enabling boxes 
+
+create symlink of a box from ./available to ./enabled e.g.
+to enable drools app one would do following, note the '../'
+```
+[vmindru@vmutil vmdefs]$ ln -s ../available/centos6-drools-guest/ ./enabled/
```

Comment:
I think this should be `ln -s available/centos6-drools-guest/ enabled/` since I'd assume everything to be relative to the `README.md` file.